### ISSUE TITLE
metaMDS(..., engine ="isoMDS") produces error

### ISSUE THREAD

**Issue by martinmodrak** [vegandevs/vegan#310]
I have frequent convergence problems with the default setting of `metaMDS` so I wanted to try `isoMDS` instead of the default, but the code does not complete. Reproducible example follows:

```r
library(vegan)
library(MASS)
data(varespec)
metaMDS(varespec)  #Completes, but does not converge
metaMDS(varespec, engine = "isoMDS")  #Does not complete
```
The error from the last line is
```
Error in out$dist/scl : non-numeric argument to binary operator
```

I am using vegan 2.5-4 with R 3.5.3 on Windows

**Comment by jarioksa** [comment]
Yes, this is a bug: `isoMDS` does not return information you need for this scaling. However, you really should not use `isoMDS` – it has several problems. However, I'll fix this.

**Comment by martinmodrak** [comment]
1) Thanks for the quick reaction. 2) I understand there are issues with `isoMDS`, I was mostly just curious what would happen 3) Sorry if I sounded like demanding support, I definitely don't need this to be fixed to carry on with life.

Thanks for the nice package!

**Comment by jarioksa** [comment]
Don't worry: we think that if we provide an option like `engine` the option should work! I am not offended or disturbed of reports of failure, but I'll do my best to fix the problems, and I'll fix this one. We just did not have a test to the `isoMDS` engine and this fault slipped through when we made an unrelated change.

About your actual problem of no-convergence: Have you read the section "Convergence Problems" in the `metaMDS` help (`?metaMDS`), and followed those procedures?
